src/apps/vpn/main.cpp

```diff
@@ -4,13 +4,16 @@
 
 #include "commandlineparser.h"
 #include "leakdetector.h"
+#include "telemetry.h"
 
 Q_DECL_EXPORT int main(int argc, char* argv[]) {
 #ifdef MZ_DEBUG
   LeakDetector leakDetector;
   Q_UNUSED(leakDetector);
 #endif
 
+  Telemetry::startTimeToMainScreenTimer();
```

Comment:
This is wrong. This should go in commandui.cpp because main is shared between the daemons, the CLI and the UI app.